README.md

```diff
@@ -1,23 +1,21 @@
 ![Logo](/.github/logo.png)
 
-analytics-android-integration-moengage
+analytics-kotlin-destination-moengage
```

Comment:
We need a note for the `moengage-segment-integration` around deprecation and link to the integration document

Reply:
Done